.tools/test/stacks/deploy.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import subprocess
+import os
+import yaml
+import time
+
+
+def run_shell_command(command, env_vars=None):
+    """Run a shell command and return its output"""
+    # Prepare the environment
+    env = os.environ.copy()
+    if env_vars:
+        env.update(env_vars)
+
+    print("COMMAND: " + command)
+    try:
+        output = subprocess.check_output(
+            command, shell=True, stderr=subprocess.STDOUT, env=env
+        )
+        print(f"Command output: {output.decode()}")
+    except subprocess.CalledProcessError as e:
+        print(f"Error executing command: {e.output.decode()}")
+
+
+def deploy_resources(account_id, account_name, dir, lang='typescript'):
+    """Deploy resources to the given account"""
+    if dir not in os.getcwd():
+        os.chdir(f"{dir}/{lang}")
+    # Get AWS tokens for the account
+    get_tokens_command = f"ada credentials update --account {account_id} --provider isengard --role weathertop-cdk-deployments --once"
+    run_shell_command(get_tokens_command)
+    export_name_command = f"export TOOL_NAME={account_name}"
```

Comment:
So it doesn't set the value? It seems to work when I use the script locally.

Reply:
It sets the value _for that shell_, but that shell will then forget it. `deploy.py` "works" because you pass `env_vars` on line 36, 32 and 33 are a very expensive noop.

Reply:
Gotcha. Removed. Good call